go.mod

```
@@ -0,0 +1,24 @@
+module fasthttpprometheus
```

That should maybe be like this, but I am not sure :)

```suggestion
module github.com/zattoo/fasthttp-prometheus
```

---

If it works as it, we can leave it as it